src/browser/HostInstaller.cpp

```diff
@@ -42,6 +42,7 @@ HostInstaller::HostInstaller()
     , TARGET_DIR_CHROME("/.config/google-chrome/NativeMessagingHosts")
     , TARGET_DIR_CHROMIUM("/.config/chromium/NativeMessagingHosts")
     , TARGET_DIR_FIREFOX("/.mozilla/native-messaging-hosts")
+    , TARGET_DIR_TOR_BROWSER("/.tor-browser/app/Browser/TorBrowser/Data/Browser/.mozilla/native-messaging-hosts")
```

Comment:
Please use the same order for the variables. Tor must come last, like you used it elsewhere, not in the middle. Also, `TARGET_DIR_TOR_BROWSER` must be defined for all OS's.

Reply:
On it, sir.

Reply:
Maybe this happens because you define `TARGET_DIR_TOR_BROWSER` in HostInstaller.h in the wrong order.

Reply:
> 
> 
> Maybe this happens because you define `TARGET_DIR_TOR_BROWSER` in HostInstaller.h in the wrong order.

:+1: Correct, in the office - going to make manual updates and hope it passes in CI.